File: server/medicore.layer/MediCore.Service/Model/HospitalOperatingHoursModel.cs
using dna.core.service.Models.Abstract;
using System.ComponentModel.DataAnnotations;

namespace MediCore.Service.Model
{
    public class HospitalOperatingHoursModel : WriteHistoryBaseModel, IModelBase
    {
        public int Id { get; set; }

        [Required(ErrorMessage = "Required")]
        public int Day { get; set; }

        [Required(ErrorMessage = "Required")]
        public int HospitalId { get; set; }

        [Required(ErrorMessage = "Required")]
        [StringLength(10, ErrorMessage = "StringLength")]
        public string StartTime { get; set; }

        [Required(ErrorMessage = "Required")]
        [StringLength(10, ErrorMessage = "StringLength")]
        public string EndTime { get; set; }

        [Required(ErrorMessage = "Required")]
        public bool IsClossed { get; set; }

        public virtual HospitalModel Hospital { get; set; }
    }
}

File: dashboard/src/app/model/hospitalOperatingHoursModel.ts
import { IEntityBase}  from './IEntityBase';
import { WriteHistoryBaseModel } from './writeHistoryBaseModel';
import { HospitalModel } from './hospitalModel'

export class HospitalOpratingHoursModel extends WriteHistoryBaseModel implements IEntityBase{
    id:number;
    day:number;
    startTime: string;
    endTime:string;
    isClossed:boolean;
    hospitalId:number;

    hospital:HospitalModel;

    constructor()
    constructor(day:number, isClossed:boolean)
    constructor(day:number, isClossed:boolean, startTime: string, endTime: string)
    constructor(day?:number, isClossed?:boolean, startTime?:string, endTime?:string){
        super();
        this.day = day;
        this.isClossed = isClossed;
        this.startTime = startTime;
        this.endTime = endTime;
        this.hospital = null;
        
    }
}